contracts/wawaka/contract-build.cmake

```diff
@@ -15,75 +15,126 @@
 IF(NOT EXISTS $ENV{PDO_HOME})
   MESSAGE(FATAL "PDO_HOME environment variable not defined")
 ENDIF()
+SET(PDO_HOME "$ENV{PDO_HOME}")
+
+IF(NOT EXISTS $ENV{PDO_SOURCE_ROOT})
+  MESSAGE(FATAL "PDO_SOURCE_ROOT environment variable not defined")
+ENDIF()
+SET(PDO_SOURCE_ROOT $ENV{PDO_SOURCE_ROOT})
+SET(WAWAKA_ROOT ${PDO_SOURCE_ROOT}/contracts/wawaka)
+
+IF (NOT DEFINED ENV{WASM_SRC})
+  MESSAGE(FATAL_ERROR "WASM_MEM_CONFIG environment variable not defined!")
+ENDIF()
+SET(WASM_SRC "$ENV{WASM_SRC}")
 
 IF (NOT DEFINED ENV{WASM_MEM_CONFIG})
   MESSAGE(FATAL_ERROR "WASM_MEM_CONFIG environment variable not defined!")
 ENDIF()
+SET(WASM_MEM_CONFIG "$ENV{WASM_MEM_CONFIG}")
+
+IF (NOT DEFINED ENV{WASI_SDK_DIR})
```

Comment:
We should probably document this variable to docs/environment.md, no?

Reply:
removed the requirement for the environment variable.